Automattic/themes
# Blockbase: initial dotorg upload version

This is a version of Blockbase that adds the required function calls and files to get it to pass the automated theme check.

The requirements to get Mayland-blocks and Seedlet-blocks have also been added to this branch.

**To test**

- Check out this PR
- From the root themes folder, run `./package-dotorg.sh blockbase`
- Locate the generated zip within the blockbase folder. Upload and activate it
- Install and activate the automated [theme check plugin](https://wordpress.org/plugins/theme-check/)
- From wp-admin, visit Appearance > Theme Check and run the theme check
- Verify no Required errors are logged: 

<img width="1392" alt="Screen Shot 2021-05-25 at 1 58 21 PM" src="https://user-images.githubusercontent.com/5375500/119546589-ed368400-bd48-11eb-9aac-c08e42ddedcf.png">
